docs/corpus.markdown

```diff
@@ -1,45 +1,166 @@
+interface. The root directory for a corpus should contain a Python module named
+`driver.py` that defines the QuantGov corpus driver for that corpus.
+Additionally, a corpus can contain a file called `metadata.csv` that contains
+any additional information about individual documents that may be relevant. For
+example, the `metadata.csv` generated by the CFR corpus includes which agency
+and department authored each individual CFR part.
 
--   A `makefile` to manage the workflow of the corpus
--   A module named `driver.py` that implements the corpus driver interface (see below)
--   A subdirectory named `scripts` containing the scripts needed to obtain, organize, and prepare the documents that make up the corpus, as well as to generate any metadata for the corpus.
--   A subdirectory named `data` that holds any intermediate data generated during preparation of the corpus.
+## Starting a New Corpus
 
-One file that each corpus should generate is a csv file in the `data` directory named `metadata.csv`. This file should contain any additional information about individual documents that may be relevant. For example, the `metadata.csv` generated by the CFR corpus includes which agency and department authored each individual CFR part, as well as the restriction and word count for each part.
+The fastest way to start a new corpus is to use the command
+`quantgov start corpus NAME` where NAME is the name for the corpus. This
+command will copy the skeleton corpus from
+<https://github.org/quantgov/corpus>. The skeleton corpus implements a
+`RecursiveDirectoryCorpusDriver` (see below), and expects a set of text files
+in a directory `data/clean` relative to the corpus base directory.
 
 ## The Corpus Driver Interface
 
-Each corpus should contain a python module named `driver.py`. This driver serves two important functions. First it specifies how the corpus should be indexed. An index is one or more values that, taken together, uniquely identify each document in the corpus. An index can be as simple as an id number, or it can be more descriptive. For the CFR corpus, each document, representing a single subdivision called a part, is represented by three pieces of metadata: the year of the CFR edition that contains it, the title that contains it, and the part number.
+Each corpus should contain a python module named `driver.py`. This driver
+serves two important functions. First it specifies how the corpus should be
+indexed. An index is one or more values that, taken together, uniquely identify
+each document in the corpus. An index can be as simple as an id number, or it
+can be more descriptive. For the CFR corpus, each document, representing a
+single subdivision called a part, is represented by three pieces of metadata:
+the year of the CFR edition in which the part was printed, the to which the
+part belongs, and the part number.
+
+The `driver.py` file should define a variable called `driver` which holds the
```

Comment:
This paragraph on the driver is slightly confusing to me. What is the driver object in this case?

Reply:
I know we give examples below, but maybe just a sentence on what a driver object does generally.

Reply:
Updated to clarify.